client/src/Movies/MovieList.js

```diff
@@ -1,5 +1,6 @@
 import React, { useState, useEffect } from 'react';
 import axios from 'axios';
+import { NavLink } from "react-router-dom";
```

Comment:
NavLink is mainly used for navigation links, such as the ones at the top of a page, Link however, gives us the capability of linking generally.